docs/content/doc/packages/overview.en-us.md

```diff
@@ -38,6 +38,28 @@ The following package managers are currently supported:
 
 **The following paragraphs only apply if Packages are not globally disabled!**
 
+## Repository-Packages
+A package always belongs to an owner (a user or organisation), not a repo.
+To link an (already uploaded) package to a repository, open the settings page
+on that package and choose a repository to link this package to.
+The entire package will be linked, not just a single version.
+
+Linking a package results in showing that package in the repository's package list,
+and shows a link to the repository on the package site (as well as a link to the repo issues).
+
+## Access Restrictions
+
+| Package owner type | User | Organization |
+|--------------------|------|--------------|
+| **read** access    | public, if user is marked not private, otherwise user only | public, if org is public, otherwise org members only |
```

Comment:
What do you mean with `if user is not marked private`? Do you mean
```suggestion
| **read** access    | public, if the repo is public, otherwise user only | public, if org is public, otherwise org members only |
```

Reply:
No, I'm talking about users:
![Screenshot_20220401_142345](https://user-images.githubusercontent.com/7880552/161262589-f36e920f-dd94-4921-b74d-65fcf5f08d47.png)

AFAIK, linked repos have no impact on ACLs, though my understanding is limited. Thus my review request to @KN4CK3R